Environments.java

```diff
@@ -0,0 +1,8 @@
+public class Environment  {
+  
+   enum envirTypes {
```

Comment:
You'll have to make this public and static to allow other classes to use it.
```suggestion
   public static enum envirTypes {
```